docs/configure/outputs/logstash.asciidoc

```diff
@@ -5,6 +5,12 @@
 <titleabbrev>{ls}</titleabbrev>
 ++++
 
+****
+image:./binary-yes-fm-no.svg[supported deployment methods]
+
+The {ls} output is not yet supported by {fleet}-managed APM Server.
+****
+
```

Comment:
Please change `Kibana output` to `Kibana configuration` in https://github.com/elastic/apm-server/pull/10842/files#diff-93a49822f5cc2506c396df37910cb6d828a9059a6fb56c12d711af977b4de355R48

Reply:
done